webapp/components/test-file-results.html

```diff
@@ -108,19 +110,19 @@
           }
           testRuns[i].subtests = {};
           testRuns[i].subtests['STATUS'] = { status: resultData.status };
-          if (['OK', 'ERROR'].includes(resultData.status)) {
-            isTestHarness = true;
-          }
-
-          for (let subtestResult of resultData.subtests) {
-            testRuns[i].subtests[subtestResult.name] = subtestResult;
-            if (!subtestNames.includes(subtestResult.name)) {
-              subtestNames.push(subtestResult.name);
+          if (resultData.subtests && resultData.subtests.length > 0) {
+            // Since we spoof the file-level status also as a "subtest", we
```

Comment:
test-level. And maybe comment here that this is still imperfect, since a testharness.js test that throws an exception before registering the first test will still say "Test status" instead of "Harness status".